lib/active_merchant/billing/gateways/mundipagg.rb

```diff
@@ -128,8 +128,8 @@ def add_billing_address(post, type, options)
       def add_shipping_address(post, options)
         if address = options[:shipping_address]
           post[:address] = {}
-          post[:address][:street] = address[:address1].match(/\D+/)[0].strip if address[:address1]
-          post[:address][:number] = address[:address1].match(/\d+/)[0] if address[:address1]
+          post[:address][:street] = address[:address1].match(/\D+/)[0].strip if address[:address1].match(/\D+/) && address[:address1]
+          post[:address][:number] = address[:address1].match(/\d+/)[0] if address[:address1].match(/\d+/) && address[:address1]
```

Comment:
Should the check that `address[:address1]` is truthy occur before the match?

Reply:
Updated.